src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -1183,8 +1215,11 @@ End Module
             End Using
         End Function
 
+        ' The test maybe flaky. Observed some failure and recovers the next run.
         <WorkItem(547287, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/547287")>
-        <MemberData(NameOf(AllCompletionImplementations))> <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
+        <InlineData(CompletionImplementation.Legacy)>
+        <InlineData(CompletionImplementation.Modern, Skip:="https://github.com/dotnet/roslyn/issues/30250")> ' Flaky
```

Comment:
this is terrifying.  if we have flaky tests, that's a *severe* issue that we have to resolve.

Reply:
yes, we have to resolve this before making the feature available.

---
In reply to: [222523484](https://github.com/dotnet/roslyn/pull/29016#discussion_r222523484) [](ancestors = 222523484)